src/views/SystemIntake/RequestDetails/index.tsx

```diff
@@ -1,5 +1,206 @@
 import React from 'react';
+import { Field, FormikProps } from 'formik';
+import { SystemIntakeForm } from '../../../types/systemIntake';
```

Comment:
We setup the project to be able to absolutely resolve imports from `src` so this should be able to be written as `types/systemIntake`.

This will clean up file structure, especially if we nest deeper. I understand that Goland auto imports these files, so there's probably a setting to have it import absolutely.